src/Control/Carrier/Class.hs-boot

```diff
@@ -0,0 +1,9 @@
+{-# LANGUAGE FunctionalDependencies #-}
+module Control.Carrier.Class
+( Carrier(..)
+) where
+
+import Control.Effect.Class
```

Comment:
In an effort to fix the aforementioned problem where `Carrier (Reader r) ((->) r)` isn’t available in e.g. `Control.Effect.Reader`, I attempted to duplicate the imports and instances here in the `.hs-boot` file, but it was not successful.